extensions/objects/templates/coord_two_dim_editor_directive.html

```diff
@@ -1,7 +1,15 @@
-<div>
-  <div ui-map="map" ui-options="mapOptions"
-       style="width: 100%; height: 400px; max-width: 600px; max-height: 400px; margin: auto"
-       ui-event="{'map-click': 'registerClick($event, $params)'}">
-  </div>
-  <div ui-map-marker="mapMarker"></div>
+<div class="oppia-coord-two-dim-editor-container">
+  <leaflet id="coordTwoDimEditor" lf-center="mapCenter" event-broadcast="mapEvents" markers="mapMarkers"></leaflet>
 </div>
+<style>
+  #coordTwoDimEditor {
```

Comment:
Could we use a class instead?

Reply:
Internally ui-leaflet use to access this id style and apply this to map container. Changing this to the class is creating some stylistic issue.

Reply:
See comment above re generating the ID and handling the CSS style separately -- is that possible? (If not, no worries, just wanted to check.)

Also for any case where you use IDs for CSS, make sure to add a comment explaining what's going on.